src/components/nav/Navbar.js

```diff
@@ -144,7 +163,11 @@ Navbar.propTypes = {
   sticky: PropTypes.string,
 
   /**
-   * Sets the color of the Navbar, options: primary, secondary, success, warning, danger, info, light.
+   * Sets the color of the Navbar. Main options are primary, light and dark, default light.
+   *
+   * You can also choose one of the other contextual classes provided by Bootstrap
+   * (secondary, success, warning, danger, info, white) or any color of your choice
+   * specified either by its HTML color name or by its hex code.
```

Comment:
Why not *or any valid CSS color of your choice (e.g. a hex code, a decimal code or a CSS color name)* ? AFAICT, this allows any valid CSS color (apart from `white`), so `rgba(1, 2, 3, 0.2)` is valid?

Reply:
Good call, added.